tauri-update/Cargo.toml

```diff
@@ -0,0 +1,22 @@
+[package]
+name = "tauri-update"
+version = "0.2.0"
+authors = ["Lucas Fernandes Gonçalves Nogueira <lucas@quasar.dev>", "Daniel Thompson-Yvetot <denjell@sfosc.org>", "Tensor Programming <tensordeveloper@gmail.com>"]
+license = "MIT"
+homepage = "https://tauri-apps.org"
+repository = "https://github.com/tauri-apps/tauri"
+description = "Make tiny, secure apps for all desktop platforms with Tauri"
```

Comment:
Crate-specific description please

Reply:
Will add description after we decide on the names for the crates (they are still tentative atm).